src/app/workflow/snapshot-exporter-modal/snaphot-exporter-modal.component.html

```diff
@@ -0,0 +1,173 @@
+              (hasZenodoToken$ | async) !== true ||
+              (action === SnapshotExporterAction.ORCID && (hasOrcidToken$ | async) !== true) ||
+              state.overall !== StepState.INITIAL
+            "
+            class="bg-accent-1-dark"
+            mat-flat-button
+            data-cy="export-button"
+            (click)="export(false)"
+            *ngIf="!promptToConfirmSnapshot"
+          >
+            {{ action === SnapshotExporterAction.DOI ? 'Request DOI' : 'Export' }}
+          </button>
+          <button
+            class="bg-accent-1-dark"
+            mat-flat-button
+            [disabled]="state.overall !== StepState.INITIAL"
+            (click)="export(true)"
+            *ngIf="promptToConfirmSnapshot"
+          >
+            Confirm
+          </button>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <ng-template #snapshotBlurb>
+    <p>
+      A snapshot freezes the workflow version, ensuring that the content of the workflow does not change, e.g., if cited in publications.
+      <strong>Important note: once a snapshot is made, the workflow version becomes uneditable, apart from some metadata.</strong>
+    </p>
+    <p>
+      You can then use this snapshot to request a DOI or export to your ORCID record.
+      <strong>Are you sure you want to snapshot this version?</strong>
+    </p>
+  </ng-template>
+
+  <ng-template #doiBlurb>
+    <p>A Digital Object Identifier (DOI) allows a workflow version to be easily cited in publications. What’s involved in the request:</p>
+    <ol>
+      <li>
+        Before a DOI can be issued, a <strong>snapshot</strong> of the workflow version must be made to ensure that the content referenced
+        by the DOI does not change.
+        <strong>Please note: a snapshot will make the workflow version uneditable, apart from some metadata.</strong>
+      </li>
+      <li>
+        Next, a DOI is requested, providing a unique identifier for this workflow version. The DOI identifier will be displayed in the
+        snapshot column of the versions table.
+      </li>
+    </ol>
+  </ng-template>
+
+  <ng-template #orcidBlurb>
+    <p>
+      Exporting to ORCID requires a Digital Object Identifier (DOI), which allows a workflow version to be easily cited in publications.
+      What’s involved:
+    </p>
+
+    <ol>
+      <li>
+        Before a DOI can be issued, a snapshot of the workflow version must be made to ensure that the content referenced by the DOI does
+        not change. Please note: a snapshot will make the workflow version uneditable, apart from some metadata.
+      </li>
+      <li>Next, a DOI is requested, providing a unique identifier for this workflow version.</li>
+      <li>Lastly, the workflow will be exported to your ORCID record.</li>
+    </ol>
+  </ng-template>
+
+  <ng-template #zenodoCheck>
+    <div *ngIf="!(hasZenodoToken$ | async)" class="alert-danger p-2" fxLayout="row" fxLayoutGap="0.625rem;" fxLayoutAlign="end top">
```

Comment:
There's about 5 of these `Async pipes should not be negated. Use (observable | async) === (false || null || undefined) to check its value instead  @angular-eslint/template/no-negated-async` in this PR. I'll fix it in my PR though since it's not much of a priority for release.

Reply:
I thought I'd fixed those, but looking at it again, I only fixed some.

Reply:
@garyluu fixed all of them per our Slack discussion. Tagging you for re-review, in case I messed something up.